content/en/docs/tasks/administer-cluster/encrypt-data.md

```diff
@@ -8,7 +8,26 @@ weight: 210
 ---
 
 <!-- overview -->
-This page shows how to enable and configure encryption of secret data at rest.
+
+Some of the APIs in Kubernetes, such as {{< glossary_tooltip text="Secret" term_id="secret" >}},
```

Comment:
All REST APIs support encryption at rest, not just 'some'.

Reply:
Are we sure? Maybe “Every API that can persist data?”

Reply:
I reworded. How's that?

Reply:
> Are we sure? Maybe “Every API that can persist data?”

Yeah we test and support wildcards so `*.*` will encrypt every REST resource.